Studio/src/Application/Visualization/ShapeWorksStudioApp.cpp

```diff
@@ -765,7 +765,7 @@ void ShapeWorksStudioApp::handle_progress(int value)
   if (value < 100) {
     this->progress_bar_->setVisible(true);
     this->progress_bar_->setValue(value);
-    this->disable_all_actions();
+    //this->disable_all_actions();
```

Comment:
?

Reply:
The commit message for this probably didn't make it clear.  I decided we no longer need to disable the actions such as "save", etc just because something is going on the progressbar.  We have other things using the progress bar such as the mesh warping.  I'll add to the cleanup commits along with conflict resolution.